plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -275,6 +275,26 @@ public class PostgreSqlClient
         return FULL_PUSHDOWN.apply(session, simplifiedDomain);
     };
 
+    private static final ColumnMapping POSTGRESQL_MERGE_ROW_ID_COLUMN_MAPPING = ColumnMapping.sliceMapping(
```

Comment:
The `ctid` is not stable while concurrent update. we need use the primary key or unique key.

Reply:
Revert the impl that rely on the `ctid`, instead, using the primary keys, it would be more general but for some tables  that not defined primarykeys might not able to perform merge.